src/rageshake/rageshake.ts

```diff
@@ -444,6 +444,11 @@ export function init(setUpPersistence = true): Promise<void> {
     global.mx_rage_logger = new ConsoleLogger();
     global.mx_rage_logger.monkeyPatch(window.console);
 
+    // log unhandled rejections in the rageshake
+    window.addEventListener("unhandledrejection", (event) => {
+        logger.error(`Unhandled promise rejection: ${event.reason}`);
```

Comment:
it feels odd that this indirects via the matrix-js-sdk `logger` rather than going straight to `global.mx_rage_logger`.